src/libs/actions/IOU.js

```diff
@@ -563,6 +583,15 @@ function createSplitsAndOnyxData(participants, currentUserLogin, currentUserAcco
             '',
             oneOnOneIOUReport.reportID,
         );
+        // Add optimistic personal details for new participants
```

Comment:
NAB we tend to have an empty line in front of comments
```suggestion

        // Add optimistic personal details for new participants
```